LearnositySDKUnitTests/RequestAsStringUnitTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using LearnositySDK.Utils;
+using LearnositySDK.Request;
+
+
+namespace LearnositySDKUnitTests
+{
+    [TestClass]
+    public class RequestAsStringUnitTests
+    {
+        private readonly string consumerKey = "yis0TYCu7U9V4o7M";
+        private readonly string consumerSecret = "74c5fd430cf1242a527f6223aebd42d30464be22";
+        private readonly string domain = "localhost";
+
+        [TestMethod]
+        public void AuthorApiItemListAsString()
+        {
+            // Arrange
+            string service = "author";
+            JsonObject security = GetSecurityObj();
+            string secret = consumerSecret;
+            JsonObject request = new JsonObject();
+
+            request.set("mode", "item_list");
+
+            JsonObject config = new JsonObject();
+            JsonObject config_item_list = new JsonObject();
+            JsonObject config_item_list_item = new JsonObject();
+            config_item_list_item.set("status", true);
+            config_item_list.set("item", config_item_list_item);
+            config.set("item_list", config_item_list);
+            request.set("config", config);
+
+            JsonObject user = new JsonObject();
+            user.set("id", "walterwhite");
+            user.set("firstname", "walter");
+            user.set("lastname", "white");
+            request.set("user", user);
+
+            // Act
+            Init initAsString = new Init(service, security.toJson(), secret, request.toJson());
+            string signedRequestAsString = initAsString.generate();
+
+            Init initAsObject = new Init(service, security, secret, request);
+            string signedRequestAsObject = initAsObject.generate();
+
+            // Assert
+            Assert.AreEqual(
+                signedRequestAsString,
+                signedRequestAsObject
```

Comment:
Both requests should actually differ, as `signedRequestAsString` should have the request quoted.

Essentially, when you generate the page for the client, in the first case, the generated host page will contain

```
LearnosityApp.init("{ \"this_request\": \"is a string for JS\" }");
```

vs 

```
LearnosityApp.init({ "this_request": "is an object for JS" });
```

Reply:
@shtrom I believe the result here _should_ be the same, whether you construct the Init object with a string or an object does not change the escaping pattern used.